docs/src/content/docs/getting-started.mdx

```diff
@@ -45,24 +45,56 @@ Try Starlight in your browser:
 [open the template on StackBlitz](https://stackblitz.com/github/withastro/starlight/tree/main/examples/basics).
 :::
 
-## Creating content with Starlight
+### Start the development server
 
-Starlight is ready for you to add new content, or bring your existing files!
+When working locally, [Astro’s development server](https://docs.astro.build/en/reference/cli-reference/#astro-dev) lets you preview your work and automatically refreshes your browser when you make changes.
 
-### File formats
+Inside your project directory, run the following command to start the development server:
 
-Starlight supports authoring content in Markdown and MDX. (You can add support for Markdoc by installing the experimental [Astro Markdoc integration](https://docs.astro.build/en/guides/integrations-guide/markdoc/).)
+<Tabs>
+<TabItem label="npm">
 
-### Add pages
+```sh
+npm run dev
+```
 
-Add new pages to your site automatically by creating `.md` or `.mdx` files in `src/content/docs/`. Add sub-folders to organize your files, and to create multiple path segments:
+</TabItem>
+<TabItem label="pnpm">
 
+```sh
+pnpm dev
 ```
-src/content/docs/hello-world.md => your-site.com/hello-world
-src/content/docs/guides/faq.md => your-site.com/guides/faq
+
+</TabItem>
+<TabItem label="Yarn">
+
+```sh
+yarn dev
 ```
 
-### Type-safe frontmatter
+</TabItem>
+</Tabs>
+
+This will log a message to your terminal with the URL of your local preview.
+Open this URL to start browsing your site.
+
+### Add content
+
+Starlight is ready for you to add new content, or bring your existing files!
+
+#### File formats
+
+Starlight supports authoring content in Markdown and MDX with no configuration required.
+You can add support for Markdoc by installing the experimental [Astro Markdoc integration](https://docs.astro.build/en/guides/integrations-guide/markdoc/).
+
+#### Add pages
+
+Add new pages to your site by creating `.md` or `.mdx` files in `src/content/docs/`.
+Use sub-folders to organize your files and to create multiple path segments.
+
+For example, a file at `src/content/docs/hello-world.md` will be published at `your-site.com/hello-world` and a file at `src/content/docs/guides/faq.md` will be published at `your-site.com/guides/faq`.
```

Comment:
This is fine! Just thinking: Just because this visually isn't as easy to make the connection between file paths and URLs, is there maybe a way to use the file tree component here to show some sample files in a hierarchy, and then say something like:

For example, this file structure will publish pages at the following URLs:

Reply:
I did think of this, but worried it might take up more space than required. I could also revert to the previous style, I just wasn’t 100% sure the `path => your-site.com/path` thing was as clear as spelling it out.

Let me try the file tree version though, it would probably help to reinforce the subdirectories especially.